xml/System.Data.Spatial/DbGeometry.xml

```diff
@@ -838,9 +838,9 @@
         <Parameter Name="index" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="index">The index.</param>
-        <summary>Returns an interior ring from this DbGeometry value at a specific position, if it represents a polygon. &lt;param name="index"&gt;The position within this geometry value from which the interior ring should be taken.&lt;/param&gt;&lt;returns&gt;The interior ring in this geometry value at the specified position, if it represents a polygon; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <returns>An interior ring from this DbGeometry value at a specific position.</returns>
+        <param name="index">The position within this geometry value from which the interior ring should be taken.</param>
+        <summary>Returns an interior ring from this DbGeometry value at a specific position, if it represents a polygon.</summary>
+        <returns>The interior ring in this geometry value at the specified position, if it represents a polygon; otherwise <see langword="null" />.</returns>
```

Comment:
```suggestion
        <returns>The interior ring in this geometry value at the specified position, if it represents a polygon; otherwise, <see langword="null" />.</returns>
```